SQL/0000-00-02-Permission.sql

```diff
@@ -33,7 +33,7 @@ CREATE TABLE `permissions` (
       'View/Create/Edit',
       'Create/Edit',
       'Edit/Upload',
-      'Edit/Upload/Delete'),
```

Comment:
I don't think we want to change this? This is the generic category used for all module permissions.

@ridz1208

Reply:
@racostas @driusan I'm okai with changing it if it's the only module using it.

Or, to be safe, just create a new one

Reply:
@driusan , @ridz1208 I'm fine with both solutions. Here the considerations:
1. The only module that is using this permission it's in facts publication.
2. We can create a new permission and leave the old one, just the already existing permission will be there but not used in any other part.
3. We should consider also that some projects might be using this permission (projects that we might not aware)

I just want to reconfirm with you. After this note do we want to leave the old permission there and create a new one or should we modify it since it's "the only module" that its using it ?

Reply:
@racostas Create a new one please

Reply:
great !